graylog2-web-interface/src/stores/users/CurrentUserStore.jsx

```diff
@@ -1,13 +1,14 @@
 import Reflux from 'reflux';
 
-import URLUtils from 'util/URLUtils';
+import { qualifyUrl } from 'util/URLUtils';
 import fetch from 'logic/rest/FetchProvider';
 import ApiRoutes from 'routing/ApiRoutes';
 
 import CombinedProvider from 'injection/CombinedProvider';
 
 const { SessionStore, SessionActions } = CombinedProvider.get('Session');
 const { StartpageStore } = CombinedProvider.get('Startpage');
+const { PreferencesActions } = CombinedProvider.get('Preferences');
```

Comment:
Out of interest, why are you using the `CombinedProvider` here as well? I thought it is the recommended to use the direct import, when possible.

Reply:
It is recommended (and factually the only way possible) for stores/actions which are not exported using `singleton`. For all other stores we need to go through the `CombinedProvider` to force core and plugins getting the same instance of the store (and actions).